### PR TITLE
Use json-c uint64_t functions

### PR DESCRIPTION
**Pull request by Moiman**
Needs at least json-c-0.14

Closes: https://github.com/ascent12/drm_info/issues/52

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -12,7 +12,7 @@ cc = meson.get_compiler('c')
 
 add_project_arguments('-D_POSIX_C_SOURCE=200809L', language: 'c')
 
-jsonc = dependency('json-c', version: '>=0.13', fallback: ['json-c', 'json_c'])
+jsonc = dependency('json-c', version: '>=0.14', fallback: ['json-c', 'json_c'])
```
Comment (emersion) [inline meson.build:15]: I think it's fine to bump the version. Debian stable has 0.12.1 anyways, so the subproject fallback is already required on these systems.

File: pretty.c [hunk 2]
```diff
@@ -393,20 +388,20 @@ static void print_fb(struct json_object *obj, const char *prefix)
 
 	if (has_legacy) {
 		printf("%s" L_VAL "Pitch: %"PRIu32"\n", prefix,
-			(uint32_t)get_object_uint64(pitch_obj));
+			(uint32_t)json_object_get_uint64(pitch_obj));
```
Comment (emersion) [inline pretty.c:391]: Is there `json_object_get_uint32`? If so, we could just use that.

Comment (Moiman) [reply pretty.c:391]: There is no json_object_get_uint32.

**Comment by emersion** [comment]
Maybe we can get rid of `new_json_object_uint64` as well?

**Comment by Moiman** [comment]
> Maybe we can get rid of `new_json_object_uint64` as well?

Yes. I updated to remove it.

**Comment by emersion** [comment]
The CI fails because `subprojects/json-c.wrap` needs to be updated to >= 0.14 for systems which don't have a recent enough json-c version. Currently the subproject fetches 0.13.

**Comment by emersion** [comment]
I've updated this branch to use an updated wrap, but I'm running into more issues. Ref https://github.com/mesonbuild/wrapdb/pull/679

**Pull request merged**